src/main/java/icons/PluginIcons.java

```diff
@@ -0,0 +1,18 @@
+package icons;
```

Comment:
Could this be in our "own" package or doesn't that work?

Reply:
sorry, this [must be explicitly the way it is](https://www.jetbrains.org/intellij/sdk/docs/reference_guide/work_with_icons_and_images.html#how-to-organize-and-how-to-use-icons) :hushed: :

"... define a class or an interface with icon constants in a top-level package called `icons`"